### DISCUSSION TITLE
Toggle ignore bindings / passthrough

### DISCUSSION THREAD

**Discussion by exic** [glzr-io/glazewm#319] [Ideas]
Since we can't use Win as mod key, it would be nice to sometimes switch off bindings which collide with bindings of other apps. So my suggestion is to add a passthrough toggle, which keeps listening only for "reactivate" and not catch other bindings while in this mode:

Press Alt+1 (is bound to `focus workspace 1`): GlazeWM focusses workspace 1
Press Alt+i (is bound to suggested `passthrough`) GlazeWM switches mode
Press Alt+2: Alt+2 is passed to current application (e.g. tmux selects window 2..)
Press Alt+i: GlazeWM reenables other bindings
Press Alt+2: GlazeWM focusses workspace 2

Maybe this can already be achieved with binding modes?

Same request, I think: https://github.com/lars-berger/GlazeWM/issues/321

**Comment by exic** [comment]
It's actually solved by using a binding mode 👍

**Comment by Archie-2021** [comment]
> Since we can't use Win as mod key, it would be nice to sometimes switch off bindings which collide with bindings of other apps. So my suggestion is to add a passthrough toggle, which keeps listening only for "reactivate" and not catch other bindings while in this mode:
> 
> Press Alt+1 (is bound to `focus workspace 1`): GlazeWM focusses workspace 1 Press Alt+i (is bound to suggested `passthrough`) GlazeWM switches mode Press Alt+2: Alt+2 is passed to current application (e.g. tmux selects window 2..) Press Alt+i: GlazeWM reenables other bindings Press Alt+2: GlazeWM focusses workspace 2
> 
> Maybe this can already be achieved with binding modes?
> 
> Same request, I think: #321

Why it is not possible to bind keys to Win instead of Alt. I really hope to do that. I am coming from Linux and so used to have Super key as my personal key that doesn't conflicts with anyother app.
Is the issue with Windows not letting go the keybindings or it is a limitation on this application?

**Reply by exic** [reply]
Windows has e.g. Win+L hardcoded to lock screen 😞 so it's a Windows not a GlazeWM limitation.

**Reply by Archie-2021** [reply]
> Windows has e.g. Win+L hardcoded to lock screen 😞 so it's a Windows not a GlazeWM limitation.

Yeah, I am running Linux in VirtualBox in it and Win+L is even over-written inside the VM. It seems like it being the worse case. But I hope the other keys could be changes by disabling useless programs MS binded them.
